integration_test/start.test.ts

```diff
@@ -0,0 +1,212 @@
+    new BigNumber(0),
+    new BigNumber(0),
+    new BigNumber(1),
+    '',
+  );
+
+  const reward = stakeRequest.gasPrice!.mul(stakeRequest.gasLimit!);
+  const mintedAmount = stakeRequest.amount!.sub(reward);
+
+  before(async () => {
+    const originAccounts = await originWeb3.eth.getAccounts();
+    const auxiliaryAccounts = await auxiliaryWeb3.eth.getAccounts();
+    originFunder = originAccounts[4];
+    auxiliaryFunder = auxiliaryAccounts[6];
+    utils = new Utils(originWeb3, auxiliaryWeb3, originFunder, auxiliaryFunder, mosaicConfig);
+
+    // Initializing facilitator config.
+    execSync(facilitatorInit, { stdio: outputOptions });
+  });
+
+  it('fund staker', async () => {
+    simpleTokenInstance = utils.getSimpleTokenInstance();
+    stakerAccount = originWeb3.eth.accounts.create('simpletoken');
+    originWeb3.eth.accounts.wallet.add(stakerAccount);
+    stakeRequest.beneficiary = stakerAccount.address;
+
+    const transferRawTx: TransactionObject<boolean> = await simpleTokenInstance.methods.transfer(
+      stakerAccount.address,
+      stakerOSTBalance,
+    );
+
+    const transferReceipt = await utils.sendTransaction(
+      transferRawTx,
+      {
+        from: originFunder,
+        gasPrice: await originWeb3.eth.getGasPrice(),
+      },
+    );
+    await utils.verifyERC20Transfer(transferReceipt, stakerAccount.address, stakerOSTBalance);
+
+    await utils.fundETHOnOrigin(stakerAccount.address);
+  });
+
+  it('funding origin and aux workers', async () => {
+    facilitatorConfig = FacilitatorConfig.fromChain(auxChainId);
+    originWorker = facilitatorConfig.chains[facilitatorConfig.originChain].worker;
+    const auxiliaryWorker = facilitatorConfig.chains[facilitatorConfig.auxChainId].worker;
+    await utils.fundETHOnOrigin(originWorker);
+    await utils.fundOSTPrimeOnAuxiliary(auxiliaryWorker);
+    const originWorkerExport = exportWorkerPrefix + originWorker;
+    auxWorkerExport = exportWorkerPrefix + auxiliaryWorker;
+    process.env[originWorkerExport] = 'origin';
+    process.env[auxWorkerExport] = 'auxiliary';
+
+    const workerOSTBalance = 500;
+
+    const transferRawTx: TransactionObject<boolean> = await simpleTokenInstance.methods.transfer(
+      originWorker,
+      workerOSTBalance,
+    );
+    const transferReceipt = await utils.sendTransaction(
+      transferRawTx,
+      {
+        from: originFunder,
+        gasPrice: await originWeb3.eth.getGasPrice(),
+      },
+    );
+    await utils.verifyERC20Transfer(transferReceipt, originWorker, workerOSTBalance);
+  });
+
+  it('whitelist origin worker', async () => {
+    await utils.whitelistOriginWorker(
+      originWorker,
+      100000000000,
+    );
+  });
+
+  it('facilitator start', async () => {
+    spawn(
+      facilitatorStart,
+      { stdio: outputOptions },
+    );
+  });
+
+  it('request stake', async () => {
+    stakeRequest.staker = stakerAccount.address;
+    const ostComposerInstance = await utils.getOSTComposerInstance();
```

Comment:
There are many un-unused await. Please double check.
```suggestion
    const ostComposerInstance = utils.getOSTComposerInstance();
```